src/styles/banner.css

```diff
@@ -1,5 +1,5 @@
 @media (max-width: 63.9375rem) {
-  .has-banner [role=application] .o-layout--2panes:before {
+  .has-banner [role=application] .o-layout-2panes:before {
```

Comment:
`o-layout--2panes` was removed in favour of `o-layout-2panes` in Cozy UI a few versions back.